test/minErrSpec.js

```diff
@@ -9,6 +9,11 @@ describe('minErr', function() {
   var emptyTestError = minErr(),
     testError = minErr('test');
 
+  var originalObjectMaxDepthInErrorMessage = objectMaxDepthInErrorMessage;
+  afterEach(function() {
+    configureErrorHandling({objectMaxDepth: originalObjectMaxDepthInErrorMessage});
```

Comment:
I probably said you could it like this (via `configureErrorHandling`), but you can actually set `objectMaxDepthInErrorMessage` directly).